src/main/java/com/sforce/cd/apexUnit/report/ApexUnitTestReportGenerator.java

```diff
@@ -99,6 +105,7 @@ public static void generateTestReport(ApexReportBean[] reportBeans, String repor
 			// Add the pass, fail counters as attributes to root element
 			rootElement.setAttribute("failures", Integer.toString(failureCount));
 			rootElement.setAttribute("tests", Integer.toString(testCount));
+			rootElement.setAttribute("time", Double.toString(totalTimeInMillis.doubleValue() / 1000));
```

Comment:
I agree with @pventurino that a NPE is not needed here. We don't have that on testCount or failureCount lines above because implicitly the values returned should not be null.